src/Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -8,16 +8,11 @@
             {{ "form_not_available"|trans({}, "SonataAdminBundle") }}
         </div>
     {% else %}
-        <form
-              {% if sonata_config.getOption('form_type') == 'horizontal' %}class="form-horizontal"{% endif %}
-              role="form"
-              action="{% block sonata_form_action_url %}{{ admin.generateUrl(url, {(admin.idParameter): objectId, 'uniqid': admin.uniqid, 'subclass': app.request.get('subclass')}) }}{% endblock %}"
-              {% if form.vars.multipart %} enctype="multipart/form-data"{% endif %}
-              method="POST"
-              {% if not sonata_config.getOption('html5_validate') %}novalidate="novalidate"{% endif %}
-              {% block sonata_form_attributes %}{% endblock %}
```

Comment:
we're losing this block

Reply:
We cannot merge the content of the block with the `form_attr`. 

Maybe we can risk this special kind of BC?